redfish-core/lib/account_service.hpp

```diff
@@ -2574,6 +2574,14 @@ inline void
             return;
         }
 
+        // Read only users should not be allowed to bypass self
+        if (mfaBypass)
```

Comment:
how does this check decides if its read only privilege or admin privilege user or not?
this check can hit with configure self privilege also right?